### PR TITLE
CO-105 - Remove unused code  module-certificate

### PR DESCRIPTION
**Pull request by Ekta29yadav**
<table>
<td>
  <a href="https://project-sunbird.atlassian.net/browse/CO-105" title="CO-105" target="_blank">Minor-EnhancementCO-105</a>  Remove unused and redundant code from Reference web app
  </td></table>
  <br />
 

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

### Type of change

Please choose appropriate options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes in the below checkboxes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Ran Test A
- [ ] Ran Test B

**Test Configuration**:
* Software versions: Angular 8, Nodejs 12.16.1
* Hardware versions:

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by arun1205 — COMMENTED** [review]
Remove the commented lines if they are not used at all

File: src/app/client/src/app/modules/workspace/components/index.ts [hunk 1]
```diff
@@ -14,18 +14,18 @@ export * from './data-driven/data-driven.component';
 export * from './content-creation-default-template/content-creation-default-template.component';
 export * from './limited-published/limited-published.component';
 export * from './flagged/flagged.component';
-export * from './batch-list/batch-list.component';
+// export * from './batch-list/batch-list.component';
```
Comment (arun1205) [inline src/app/client/src/app/modules/workspace/components/index.ts:17]: Plz remove the commented lines from the file

File: src/app/client/src/app/plugins/profile/components/index.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
-export * from './profile-badge/profile-badge.component';
-export * from './profile-page/profile-page.component';
+// export * from './profile-badge/profile-badge.component';
+// export * from './profile-page/profile-page.component';
```
Comment (arun1205) [inline src/app/client/src/app/plugins/profile/components/index.ts:2]: Plz remove the commented lines from the file

Comment (Ekta29yadav) [reply src/app/client/src/app/plugins/profile/components/index.ts:2]: done

File: src/app/client/src/app/modules/workspace/components/index.ts (hunk 1 above)
Comment (Ekta29yadav) [reply src/app/client/src/app/modules/workspace/components/index.ts]: done

**Pull request closed without merging**